src/lab8-tests.md

```diff
@@ -172,6 +172,7 @@ Testing focus
 
     Clicking back on the content area unfocuses it
     >>> browser.handle_click(test.Event(200, 200))
+    Ignoring HTML contents inside button
```

Comment:
Why do these change?

Reply:
Because calling `render` can cause `InputLayout.paint` to be called, and
we're now calling `render` even if nothing was hit tested (to cover situations
where you have an input focused and then click in an empty location on the
page). Line 287 in the code.